web/packages/teleport/src/services/api/api.js

```diff
@@ -30,6 +30,25 @@ const api = {
     });
   },
 
+  postFormData(url, formData) {
```

Comment:
Would it be better that we include `csrf` token here, instead of how we include them directly in the [SubmittablePluginForm](https://github.com/gravitational/teleport.e/blob/9c11e4684305e440d7299b32540a952ae99d98dc/web/teleport/src/Integrations/IntegrationEnroll/PluginEnroll/SubmittablePluginForm.tsx#L64):  `<input type="hidden" name="csrf_token" value={getXCSRFToken()} />` ?

Reply:
all fetch request already sets a custom header with the token: https://github.com/gravitational/teleport/blob/4ec3b4fd4a54a9a66605f22422a8d24126056540/web/packages/teleport/src/services/api/api.js#L102, so using the submittable form, we are submitting and checking it twice essentially